buildspec.pypi.yml

```diff
@@ -0,0 +1,38 @@
+version: 0.2
+
+env:
+  shell: bash
+  parameter-store:
+    PYPI_TOKEN: "pypi-token"
+
+phases:
+  install:
+    commands:
+      - echo Setting local Python versions
+      - pyenv versions | awk 'match($0, /\d\.\d+\.\d+/) { print substr($0, RSTART, RLENGTH) }' | sort -rV > .python-version
+      - echo Installing dependencies
+      - pip install poetry
+      - poetry install
+
+  build:
+    commands:
+      - python --version
+      - echo Running check-cloudformation
+      - poetry run ./copilot_helper.py check-cloudformation
+      - echo Running tests
+      - poetry run tox
+      - |
+        python --version
```

Comment:
This line is no longer needed.

```suggestion
```

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/139/commits/22ab4bf423dcd58c1b3e639e6fceaed250e85267.